client/stylesheets/_toCat.scss

```diff
@@ -4347,14 +4347,14 @@ On click the images will be displayed at normal size to complete the effect
 
 
 #profileReceivedOffersMaps {
-    margin-top: 20px;
+    margin-top: 10px;
     width: 100%;
     height: 180px;
 }
 
 
 .profileViewMeetupRight li {
-    margin: 10px 0 !important;
+    margin: 16px 0 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces